INFO 2025-01-17 03:56:58,004 kevm_pyk.summarizer - Number of opcodes: 84
INFO 2025-01-17 03:56:58,004 kevm_pyk.summarizer - Number of passed opcodes: 72
INFO 2025-01-17 03:56:58,004 kevm_pyk.summarizer - Number of todo opcodes: 12
INFO 2025-01-17 03:56:58,004 kevm_pyk.summarizer - Todo opcodes: ['PUSH', 'LOG', 'CREATE', 'CALL', 'CALLCODE', 'RETURN', 'DELEGATECALL', 'CREATE2', 'STATICCALL', 'REVERT', 'INVALID', 'SELFDESTRUCT']
INFO 2025-01-17 03:56:58,004 kevm_pyk.__main__ - summarizing ['PUSH', 'LOG', 'CREATE', 'CALL', 'CALLCODE', 'RETURN', 'DELEGATECALL', 'CREATE2', 'STATICCALL', 'REVERT', 'INVALID', 'SELFDESTRUCT']
INFO 2025-01-17 03:56:58,005 pyk.kast.outer - Loading JSON definition: /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/compiled.json
INFO 2025-01-17 03:56:58,133 pyk.kast.outer - Converting JSON definition to Kast: /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/compiled.json
INFO 2025-01-17 03:56:58,645 pyk.utils - Making directory: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/PUSH_0_SPEC/kcfg
INFO 2025-01-17 03:56:58,646 pyk.utils - Making directory: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/PUSH_0_SPEC/kcfg/nodes
INFO 2025-01-17 03:56:58,649 kevm_pyk.summarizer - Setting the type of `USEGAS_CELL` to `false` for PUSH
INFO 2025-01-17 03:56:58,650 pyk.kore.rpc - Starting KoreServer: kore-rpc-booster /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/definition.kore --module EDSL --server-port 0 --llvm-backend-library /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/llvm-library/interpreter.so --interim-simplification 25
INFO 2025-01-17 03:56:58,656 pyk.kore.rpc - [PID=3246026][stde] [proxy] Loading definition from /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/definition.kore, main module "EDSL"
INFO 2025-01-17 03:56:59,658 pyk.kore.rpc - [PID=3246026][stde] [kore][info] Reading the input file TimeSpec {sec = 0, nsec = 25463044}
INFO 2025-01-17 03:57:00,659 pyk.kore.rpc - [PID=3246026][stde] [kore][info] Parsing the file TimeSpec {sec = 0, nsec = 120}
INFO 2025-01-17 03:57:01,660 pyk.kore.rpc - [PID=3246026][stde] [kore][info] Verifying the definition TimeSpec {sec = 0, nsec = 231}
INFO 2025-01-17 03:57:02,661 pyk.kore.rpc - [PID=3246026][stde] [kore][info] Executing TimeSpec {sec = 0, nsec = 729854270}
INFO 2025-01-17 03:57:02,661 pyk.kore.rpc - [PID=3246026][stde] [kore][info] Reading the input file TimeSpec {sec = 0, nsec = 27822918}
INFO 2025-01-17 03:57:02,661 pyk.kore.rpc - [PID=3246026][stde] [kore][info] Parsing the file TimeSpec {sec = 0, nsec = 120}
INFO 2025-01-17 03:57:03,598 pyk.kore.rpc - KoreServer started: 0.0.0.0:42043, pid=3246026
INFO 2025-01-17 03:57:03,598 pyk.kore.rpc - Connecting to host: localhost:42043
INFO 2025-01-17 03:57:03,599 pyk.kore.rpc - Connected to host: localhost:42043
INFO 2025-01-17 03:57:03,599 kevm_pyk.utils - Computing definedness constraint for initial node: PUSH_0_SPEC
INFO 2025-01-17 03:57:03,612 pyk.kore.rpc - Sending request to localhost:42043: 139644415536848-001 - simplify
INFO 2025-01-17 03:57:03,662 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415536848-001
INFO 2025-01-17 03:57:03,662 pyk.kore.rpc - [PID=3246026][stde] [kore][info] Verifying the definition TimeSpec {sec = 0, nsec = 230}
INFO 2025-01-17 03:57:03,662 pyk.kore.rpc - [PID=3246026][stde] [proxy] Starting RPC server
INFO 2025-01-17 03:57:04,214 pyk.kore.rpc - Received response from localhost:42043: 139644415536848-001 - simplify
INFO 2025-01-17 03:57:04,220 pyk.kore.rpc - Sending request to localhost:42043: 139644415536848-002 - simplify
INFO 2025-01-17 03:57:04,665 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415536848-002
INFO 2025-01-17 03:57:04,886 pyk.kore.rpc - Received response from localhost:42043: 139644415536848-002 - simplify
INFO 2025-01-17 03:57:04,886 kevm_pyk.utils - Simplifying initial and target node: PUSH_0_SPEC
INFO 2025-01-17 03:57:04,891 pyk.kore.rpc - Sending request to localhost:42043: 139644415536848-003 - simplify
INFO 2025-01-17 03:57:05,471 pyk.kore.rpc - Received response from localhost:42043: 139644415536848-003 - simplify
INFO 2025-01-17 03:57:05,478 pyk.kore.rpc - Sending request to localhost:42043: 139644415536848-004 - simplify
INFO 2025-01-17 03:57:05,666 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415536848-003
INFO 2025-01-17 03:57:05,666 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415536848-004
INFO 2025-01-17 03:57:06,072 pyk.kore.rpc - Received response from localhost:42043: 139644415536848-004 - simplify
INFO 2025-01-17 03:57:06,076 pyk.proof.reachability - Wrote proof data for PUSH_0_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/PUSH_0_SPEC/proof.json
INFO 2025-01-17 03:57:06,077 pyk.kore.rpc - Connecting to host: localhost:42043
INFO 2025-01-17 03:57:06,077 pyk.kore.rpc - Connected to host: localhost:42043
INFO 2025-01-17 03:57:06,077 pyk.proof.proof - [ZJH] prover type: <class 'pyk.proof.reachability.APRProver'>
INFO 2025-01-17 03:57:06,077 pyk.proof.reachability - [ZJH] init proof
INFO 2025-01-17 03:57:06,080 pyk.kore.rpc - Sending request to localhost:42043: 139644415660496-001 - add-module
INFO 2025-01-17 03:57:06,667 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415660496-001
INFO 2025-01-17 03:57:06,933 pyk.kore.rpc - Received response from localhost:42043: 139644415660496-001 - add-module
INFO 2025-01-17 03:57:06,960 pyk.kore.rpc - Sending request to localhost:42043: 139644415660496-002 - add-module
INFO 2025-01-17 03:57:07,668 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415660496-002
INFO 2025-01-17 03:57:07,824 pyk.kore.rpc - Received response from localhost:42043: 139644415660496-002 - add-module
INFO 2025-01-17 03:57:07,825 pyk.proof.reachability - [ZJH] pending: [1]
INFO 2025-01-17 03:57:07,825 pyk.proof.reachability - [ZJH] node_id: 1
INFO 2025-01-17 03:57:07,825 pyk.proof.reachability - [ZJH] is_terminal: False
INFO 2025-01-17 03:57:07,825 pyk.proof.reachability - [ZJH] node_id: 2
INFO 2025-01-17 03:57:07,826 pyk.proof.reachability - [ZJH] is_terminal: False
INFO 2025-01-17 03:57:07,826 pyk.proof.reachability - [ZJH] pending: [1]
INFO 2025-01-17 03:57:07,826 pyk.proof.proof - [ZJH] proof initialized
INFO 2025-01-17 03:57:07,826 pyk.proof.proof - [ZJH] pending: [1]
INFO 2025-01-17 03:57:07,826 pyk.proof.reachability - Before appending step: 1
INFO 2025-01-17 03:57:07,826 pyk.kore.rpc - Connecting to host: localhost:42043
INFO 2025-01-17 03:57:07,826 pyk.proof.proof - Submitted steps for proof: PUSH_0_SPEC
INFO 2025-01-17 03:57:07,826 pyk.kore.rpc - Connected to host: localhost:42043
INFO 2025-01-17 03:57:07,857 pyk.kore.rpc - Sending request to localhost:42043: 139644415868496-001 - implies
INFO 2025-01-17 03:57:08,409 pyk.kore.rpc - Received response from localhost:42043: 139644415868496-001 - implies
INFO 2025-01-17 03:57:08,415 pyk.kore.rpc - [zjh] [kore-client] log-successful-rewrites: True
INFO 2025-01-17 03:57:08,415 pyk.kore.rpc - [zjh] [kore-client] log-failed-rewrites: True
INFO 2025-01-17 03:57:08,415 pyk.kore.rpc - Sending request to localhost:42043: 139644415868496-002 - execute
INFO 2025-01-17 03:57:08,669 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415868496-001
INFO 2025-01-17 03:57:08,669 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415868496-002
INFO 2025-01-17 03:57:11,413 pyk.kore.rpc - Received response from localhost:42043: 139644415868496-002 - execute
INFO 2025-01-17 03:57:11,414 pyk.kore.rpc - [zjh] [kore-client] result: dict_keys(['reason', 'depth', 'state', 'next-states', 'logs'])
INFO 2025-01-17 03:57:11,414 pyk.kore.rpc - [zjh] [kore-client] logs: []
INFO 2025-01-17 03:57:11,434 pyk.kcfg.kcfg - Extending current KCFG with the following: 2 branches: 1 --> [3, 4]: ['1023 <Int #sizeWordStack ( WS:WordStack , 0 )', 'notBool #sizeWordStack ( WS:WordStack , 0 ) <Int 0 andBool notBool 1023 <Int #sizeWordStack ( WS:WordStack , 0 )']
INFO 2025-01-17 03:57:11,435 pyk.proof.reachability - Wrote proof data for PUSH_0_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/PUSH_0_SPEC/proof.json
INFO 2025-01-17 03:57:11,438 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-17 03:57:11,438 pyk.proof.reachability - Before appending step: 4
INFO 2025-01-17 03:57:11,444 pyk.kore.rpc - Connecting to host: localhost:42043
INFO 2025-01-17 03:57:11,470 pyk.kore.rpc - Connected to host: localhost:42043
INFO 2025-01-17 03:57:11,471 pyk.kore.rpc - Sending request to localhost:42043: 139644415868496-003 - implies
INFO 2025-01-17 03:57:11,486 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415868496-003
INFO 2025-01-17 03:57:11,501 pyk.kore.rpc - Sending request to localhost:42043: 139644414808848-001 - implies
INFO 2025-01-17 03:57:12,141 pyk.kore.rpc - Received response from localhost:42043: 139644414808848-001 - implies
INFO 2025-01-17 03:57:12,149 pyk.kore.rpc - [zjh] [kore-client] log-successful-rewrites: True
INFO 2025-01-17 03:57:12,149 pyk.kore.rpc - [zjh] [kore-client] log-failed-rewrites: True
INFO 2025-01-17 03:57:12,149 pyk.kore.rpc - Received response from localhost:42043: 139644415868496-003 - implies
INFO 2025-01-17 03:57:12,149 pyk.kore.rpc - Sending request to localhost:42043: 139644414808848-002 - execute
INFO 2025-01-17 03:57:12,155 pyk.kore.rpc - [zjh] [kore-client] log-successful-rewrites: True
INFO 2025-01-17 03:57:12,155 pyk.kore.rpc - [zjh] [kore-client] log-failed-rewrites: True
INFO 2025-01-17 03:57:12,155 pyk.kore.rpc - Sending request to localhost:42043: 139644415868496-004 - execute
INFO 2025-01-17 03:57:12,485 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644414808848-002
INFO 2025-01-17 03:57:12,485 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415868496-004
INFO 2025-01-17 03:57:12,485 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644414808848-001
INFO 2025-01-17 03:57:20,403 pyk.kore.rpc - Received response from localhost:42043: 139644415868496-004 - execute
INFO 2025-01-17 03:57:20,407 pyk.kore.rpc - [zjh] [kore-client] result: dict_keys(['reason', 'depth', 'state', 'next-states', 'logs'])
INFO 2025-01-17 03:57:20,407 pyk.kore.rpc - [zjh] [kore-client] logs: [{'tag': 'rewrite', 'origin': 'kore-rpc', 'result': {'tag': 'success', 'rule-id': '8665f703a5394319f38d43434dc1e720c47c3bb1f60110b33f6cc02fa0ccd58a'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'success', 'rule-id': '7c6df1b7a841ef7cce64eaae0c332bb9383a2692d649e88e8ea49fcb7fa5d540'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'failure', 'reason': 'No applicable rules found'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'failure', 'reason': 'No applicable rules found'}}]
INFO 2025-01-17 03:57:20,460 pyk.kcfg.explore - [zjh] next_node_logs: ()
INFO 2025-01-17 03:57:20,461 pyk.kcfg.explore - [zjh] rules: []
INFO 2025-01-17 03:57:20,461 pyk.kcfg.explore - [zjh] _log_succ_rewrites: True
INFO 2025-01-17 03:57:20,462 pyk.kcfg.kcfg - Extending current KCFG with the following: basic block at depth 2: 3 --> 5
INFO 2025-01-17 03:57:20,462 pyk.proof.reachability - Wrote proof data for PUSH_0_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/PUSH_0_SPEC/proof.json
INFO 2025-01-17 03:57:20,464 pyk.proof.reachability - Before appending step: 4
INFO 2025-01-17 03:57:20,464 pyk.proof.reachability - Before appending step: 5
INFO 2025-01-17 03:57:20,467 pyk.proof.reachability - Wrote proof data for PUSH_0_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/PUSH_0_SPEC/proof.json
INFO 2025-01-17 03:57:20,469 pyk.proof.reachability - Before appending step: 4
INFO 2025-01-17 04:03:20,704 pyk.kore.rpc - Received response from localhost:42043: 139644414808848-002 - execute
INFO 2025-01-17 04:03:21,363 pyk.kore.rpc - [zjh] [kore-client] result: dict_keys(['reason', 'depth', 'state', 'next-states', 'logs'])
INFO 2025-01-17 04:03:21,363 pyk.kore.rpc - [zjh] [kore-client] logs: [{'tag': 'rewrite', 'origin': 'kore-rpc', 'result': {'tag': 'success', 'rule-id': 'a49bbf5c4b82ee9c477b266571ac6462951ccd1a83f52cbf657b2c60368c2c04'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'success', 'rule-id': 'f445bdd373dac8403ad00559717364f6630b19cbc6e53d3bbd6fa784e4089f0d'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'success', 'rule-id': '252afec90cf7fb845ffd258119c187b355a11eed9c39c5e3f4e0be6d6a362cfd'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'success', 'rule-id': '9940b671074c218a6c5f0f9e7b90d05efbcc000ee2884306b43860e14f6979e4'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'success', 'rule-id': 'a51b8eb776102d05aa610a72feb53bf9e5e6fca736f39c1c9e45c81d5db5b787'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'success', 'rule-id': 'cf46a1c55ae03d7124b8c4d35be85667c49f90ee5e2aa9e23faa9190b8e37682'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'success', 'rule-id': '27458eb47839269121ea93b912b579bddb9557fccc51e69983d94b2e6ed21335'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'failure', 'reason': 'No applicable rules found'}}, {'tag': 'rewrite', 'origin': 'booster', 'result': {'tag': 'failure', 'reason': 'No applicable rules found'}}]
INFO 2025-01-17 04:03:23,091 pyk.kcfg.explore - [zjh] next_node_logs: ()
INFO 2025-01-17 04:03:23,091 pyk.kcfg.explore - [zjh] rules: []
INFO 2025-01-17 04:03:23,091 pyk.kcfg.explore - [zjh] _log_succ_rewrites: True
INFO 2025-01-17 04:03:23,108 pyk.kcfg.kcfg - Extending current KCFG with the following: basic block at depth 7: 4 --> 6
INFO 2025-01-17 04:03:23,108 pyk.proof.reachability - Wrote proof data for PUSH_0_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/PUSH_0_SPEC/proof.json
INFO 2025-01-17 04:03:23,110 pyk.proof.reachability - Before appending step: 6
INFO 2025-01-17 04:03:23,162 pyk.kore.rpc - Sending request to localhost:42043: 139644415868496-005 - implies
INFO 2025-01-17 04:03:23,164 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415868496-005
INFO 2025-01-17 04:03:23,898 pyk.kore.rpc - Received response from localhost:42043: 139644415868496-005 - implies
INFO 2025-01-17 04:03:23,904 pyk.proof.reachability - Subsumed into target node PUSH_0_SPEC: (6, 2)
INFO 2025-01-17 04:03:23,904 pyk.proof.reachability - Wrote proof data for PUSH_0_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/PUSH_0_SPEC/proof.json
INFO 2025-01-17 04:03:23,910 pyk.kore.rpc - Sending request to localhost:42043: 139644415660496-003 - simplify
INFO 2025-01-17 04:03:24,165 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415660496-003
INFO 2025-01-17 04:03:24,515 pyk.kore.rpc - Received response from localhost:42043: 139644415660496-003 - simplify
INFO 2025-01-17 04:03:24,522 pyk.kore.rpc - Sending request to localhost:42043: 139644415660496-004 - simplify
INFO 2025-01-17 04:03:25,116 pyk.kore.rpc - Received response from localhost:42043: 139644415660496-004 - simplify
INFO 2025-01-17 04:03:25,147 pyk.kore.rpc - Sending request to localhost:42043: 139644415660496-005 - implies
INFO 2025-01-17 04:03:25,166 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415660496-004
INFO 2025-01-17 04:03:25,166 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415660496-005
INFO 2025-01-17 04:03:25,710 pyk.kore.rpc - Received response from localhost:42043: 139644415660496-005 - implies
INFO 2025-01-17 04:03:25,742 pyk.kore.rpc - Sending request to localhost:42043: 139644415660496-006 - implies
INFO 2025-01-17 04:03:26,167 pyk.kore.rpc - [PID=3246026][stde] [proxy] Processing request 139644415660496-006
INFO 2025-01-17 04:03:26,299 pyk.kore.rpc - Received response from localhost:42043: 139644415660496-006 - implies
INFO 2025-01-17 04:03:26,307 pyk.proof.reachability - Wrote proof data for PUSH_0_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/PUSH_0_SPEC/proof.json
INFO 2025-01-17 04:03:26,309 kevm_pyk.utils - Proof status PUSH_0_SPEC: ProofStatus.FAILED
INFO 2025-01-17 04:03:26,309 pyk.kore.rpc - Stopping KoreServer: 0.0.0.0:42043, pid=3246026
INFO 2025-01-17 04:03:26,386 pyk.kore.rpc - KoreServer stopped: 0.0.0.0:42043, pid=3246026
Proof timing PUSH_0_SPEC: 380.3096935749054s
